napari/layers/labels/labels.py

```diff
@@ -802,30 +802,14 @@ def _on_editable_changed(self) -> None:
             self.mode = Mode.PAN_ZOOM
             self._reset_history()
 
-    def _lookup_with_low_discrepancy_image(self, im, selected_label=None):
-        """Returns display version of im using low_discrepancy_image.
+    def _to_vispy_texture_dtype(self, data):
```

Comment:
staticmethod?

Reply:
maybe followup PR?

Reply:
ok. maybe merge this, as I use `_as_type` in #6182 so need to merge this to rename call.